.github/pull_request_template.md

```diff
@@ -17,6 +17,6 @@ when it is merged.
 
 #### Checklist
 [Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
-- [ ] PR is based off the current tip of the `next` branch
+- [ ] PR is based off the current tip of the `next` branch and targets `next`, not `master`
```

Comment:
I'd rather like to see there being two different points since users don't read the entire sentence. But, we can leave this as is for now and revisit in future if need be.